yamusic/yamusic.go

```diff
@@ -38,6 +41,10 @@ type (
 	}
 )
 
+var (
+	debl = log.New(os.Stdout, "[debug]\t", log.Ldate|log.Ltime|log.Lshortfile)
```

Comment:
Not clear name `debl`. Maybe something of `debuglog`, `deblog`, `logdebug`?
Also maybe set prefix with upper case `[DEBUG]\t`?

Reply:
Okay, I'll change name to "deblog"